acceptance-tests.md

```diff
@@ -36,26 +66,56 @@ Reference for NERC OpenStack: [https://nerc-project.github.io/nerc-docs/get-star
+            1. For a given project named `012345myproject` for example, check that the given project is listed in the `oc` CLI:
+                1. `oc get Project/012345myproject`<br/> ![Get project](img/acceptance-tests/get-project.png)
+            2. For a given namespace named `01234567-89ab-cdef-0123-456789abcdef` for example,  check that the given namespace is listed in the `oc` CLI:
+                1. `oc get Namespace/01234567-89ab-cdef-0123-456789abcdef`<br/> ![Get namespace](img/acceptance-tests/get-namespace.png)
+            3. You can explore quotas from within the Observability dashboard. For a given project named `012345myproject`, and resource `limits.cpu` or `limits.memory` for example, [Click here to check the `value` for the type=hard (max limit) and type=used (current value)](https://multicloud-console.apps.nerc-ocp-infra.rc.fas.harvard.edu/grafana/explore?orgId=1&left=%5B%22now-1h%22,%22now%22,%22Observatorium%22,%7B%22exemplar%22:true,%22expr%22:%22kube_resourcequota%7Bcluster%3D%5C%22nerc-ocp-prod%5C%22,namespace%3D%5C%2201234567-89ab-cdef-0123-456789abcdef%5C%22,resource%3D%5C%22limits.cpu%5C%22%7D%22%7D%5D).<br/> ![Observability Limits CPU](img/acceptance-tests/limits-cpu.png)
+        6. Check that an OpenShift User exists, with access to the project allocations:
+            1. For a given username of `nerc-test-account` for example, check that the given username is listed in the `oc` CLI:
+                1. `oc get User/nerc-test-account`<br/> ![NERC test account](img/acceptance-tests/managing-users-1.1.2-user-exists.png)
+            2. For a given username of `nerc-test-account` and a namespace of `01234567-89ab-cdef-0123-456789abcdef` for example, check the user's RoleBindings exist:
+                1. `oc get RoleBinding -n 01234567-89ab-cdef-0123-456789abcdef -o wide`<br/> ![NERC test account role bindings](img/acceptance-tests/nerc-test-account-RoleBindings.png)
+            3. Verify that the given username has a RoleBinding with `Role ref` of `edit`, for the project's namespaces.
+            4. Try logging into OpenShift as the user to test the Keycloak authentication.
 2. **Deactivate a project or resource allocation**
     * _As an administrator, I should be able to archive any project or resource allocation and release the resources associated with it back to the pool._
-    * <span style="text-decoration:underline;">Steps</span>
-        1. A Coldfront admin can navigate to the project
-            1. They can archive a project and expire all associated allocations by clicking ‘archive project’ by navigating to the project and clicking ‘archive project’.
-            2. They can navigate to an allocation, set the status to “Denied”, and update the allocation
+    * Steps
+        1. A ColdFront admin can navigate to the project. [See the ColdFront documentation here](https://nerc-project.github.io/nerc-docs/get-started/get-an-allocation/).
```

Comment:
Should these steps be written more actively? The heading hear is "Steps", but this isn't really a list of steps; in it's current form it's a list of criteria.